modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -5,16 +5,17 @@
 [id="metering-store-data-in-shared-volumes_{context}"]
 = Storing data in shared volumes
 
+Metering has no storage by default, but it can use any ReadWriteMany PersistentVolume or any StorageClass that provisions a ReadWriteMany PersistentVolume. A Network File System (NFS) implementation with distributed and parallel capability, such as Parallel NFS (pNFS), is supported if the NFS provider also supports the {product-title} implementation.
+
```

Comment:
I'm fine with adding this kind of configuration to our docs as an easy barrier of entry to using metering, but I would like to make it clear creating these resources should only be using when playing around with the operator and its functionality and shouldn't be used in production environment.

Reply:
I'm also not entirely sure what our recommendations are for support. Do we support third-party vendors that provide NFS implementations with the caveat that they also provide support?

Reply:
Thank you for this feedback @timflannagan1. I took the "support" language from Sergio's comment in the BZ, but can adjust depending out the Metering support agreement information.

I'll figure out wording to convey that NFS can be used to try out Metering, but that it is not recommended for production-level use.

Reply:
I would add. "In our tests, NFS and pNFS can fail to provide the performance that is needed for the operator to work appropriately, so we don't recommend its use in production".